src/app/components/Inside/Pages/Catalog/ForYou/ForYouPage.jsx

```diff
@@ -56,11 +56,11 @@ class ForYouPage extends React.Component {
     const { recentlyPlayed } = this.state;
 
     if (recentlyPlayed === false) {
-      return null;
+      return true; // No items
```

Comment:
In our render() for the ForYouPage we have this on line 200:

```
{!(recentlyPlayed && heavyRotation && recommendations) && <Loader />}
```

So if we return null for any of them it will show the loader. We must return either true or the data or it will always show this loading animation.

Reply:
Ohh okayy